java/base-client/src/main/java/no/bankaxept/epayment/client/base/accesstoken/ScheduledAccessTokenPublisher.java

```diff
@@ -29,19 +29,38 @@ public class ScheduledAccessTokenPublisher implements AccessTokenPublisher, Flow
 
     private final Queue<Flow.Subscriber<? super String>> subscribers = new LinkedBlockingQueue<>();
 
+    private String grantType;
+    private String scopes;
```

Comment:
So Thomas suggested these shouldn't just be strings, or set to any value by the user.. But I spent so long on this, I didn't really do it properly yet. These should probably be enums you give a list of..?